data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
       "Янсен",
       "Янси",
       "Ян",
-      "Ярбро"
+      "Ярбро",
+      [ "Авель", "Abel" ],
+      [ "Авраам", "Abraham" ],
+      [ "Акерс", "Akers" ],
+      [ "Аланиз", "Alaniz" ],
+      [ "Александр", "Alexander" ],
+      [ "Алли", "Alley" ],
+      [ "Диамант", "Diamond" ],
+      [ "Амайя", "Amaya" ],
+      [ "Амарал", "Amaral" ],
+      [ "Амос", "Amos" ],
+      [ "Андрей", "Andrew" ],
+      [ "Арии", "Arias" ],
+      [ "Аруда", "Arruda" ],
+      [ "Арфа", "Harp" ],
+      [ "Байнум", "Bynum" ],
+      [ "Бэкон", "Bacon" ],
+      [ "Бальдерас", "Balderas" ],
+      [ "Банкс", "Banks" ],
+      [ "Барретт", "Barrett" ],
+      [ "Барриентос", "Barrientos" ],
+      [ "Барфилд", "Barfield" ],
+      [ "Бас", "Bass" ],
+      [ "Бассетт", "Bassett" ],
+      [ "Баумана", "Bauman" ],
+      [ "Бегай", "Begay" ],
+      [ "Бейн", "Bain" ],
+      [ "Беланжер", "Belanger" ],
+      [ "Уайт", "White" ],
+      [ "Бердик", "Burdick" ],
+      [ "Берк", "Burk" ],
+      [ "Беркетт", "Burkett" ],
+      [ "Беркс", "Burks" ],
+      [ "Беркхарт", "Burkhart" ],
+      [ "Бернар", "Bernard" ],
+      [ "Бернс", "Burns" ],
+      [ "Бернье", "Bernier" ],
+      [ "Берр", "Burr" ],
+      [ "Беррис", "Burris" ],
+      [ "Берт", "Burt" ],
+      [ "Бертон", "Burton" ],
+      [ "Биби", "Beebe" ],
+      [ "Бизли", "Beasley" ],
+      [ "Бёрч", "Birch" ],
+      [ "Бланд", "Bland" ],
+      [ "Блаунт", "Blount" ],
+      [ "Блек", "Black" ],
+      [ "Блидсо", "Bledsoe" ],
+      [ "Блэкбёрн", "Blackburn" ],
+      [ "Бин", "Bean" ],
+      [ "Баттл", "Battle" ],
+      [ "Боннэр", "Bonner" ],
+      [ "Борхес", "Borges" ],
+      [ "Ботельо", "Botelho" ],
+      [ "Боуг", "Baugh" ],
+      [ "Бошан", "Beauchamp" ],
+      [ "Брайс", "Brice" ],
+      [ "Брантли", "Brantley" ],
+      [ "Бранхам", "Branham" ],
+      [ "Брасвелл", "Braswell" ],
+      [ "Брей", "Bray" ],
+      [ "Брейди", "Brady" ],
+      [ "Бренд", "Brand" ],
+      [ "Брубейкер", "Brubaker" ],
+      [ "Бруссар", "Broussard" ],
+      [ "Брэннон", "Brannon" ],
+      [ "Бургер", "Burger" ],
+      [ "Бурлесон", "Burleson" ],
+      [ "Куик", "Quick" ],
+      [ "Бьюли", "Beaulieu" ],
+      [ "Бэбб", "Babb" ],
+      [ "Бэрри", "Berry" ],
+      [ "Бэррон", "Barron" ],
+      [ "Бёрден", "Burden" ],
+      [ "Бёрдетт", "Burdette" ],
+      [ "Бёрнем", "Burnham" ],
+      [ "Вайз", "Wise" ],
+      [ "Вайзмен", "Wiseman" ],
+      [ "Валентайн", "Valentine" ],
+      [ "Ванхорн", "Vanhorn" ],
+      [ "Ватсон", "Watson" ],
```

Comment:
This one is peculiar because it's strongly, almost-irreverisbly associated with one Dr. Watson, Sherlock Holmes' expositor and the in-universe writer of the stories. Having been etched into the public mind back when it would have been transcribed as `Ватсон`, it's retained even in the latest, more-modern iterations of the stories (e.g. `Джон Ватсон` in *Sherlock* and `Джоан Ватсон` for the female counterpart in *Elementary*).

On the other hand, when not associated with the character, `Уотсон`, a phonetically-accurate spelling, is used, as in the case of [Эмма Уотсон](https://ru.wikipedia.org/wiki/%D0%A3%D0%BE%D1%82%D1%81%D0%BE%D0%BD,_%D0%AD%D0%BC%D0%BC%D0%B0), the actress who portrayed Hermione Granger in the *Harry Potter* series of films.

Given that there's no strong association with the character, I suggest we go with the latter.